### ISSUE TITLE
Some settings get lost in export

### ISSUE THREAD

**Issue by GfEW** [FossifyOrg/Gallery#133]
### Checklist

- [X] I can reproduce the bug with the latest version given [here](https://github.com/FossifyOrg/Gallery/releases/latest).
- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Gallery/issues) or [closed](https://github.com/FossifyOrg/Gallery/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Gallery/discussions) or [closed](https://github.com/FossifyOrg/Gallery/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Gallery/blob/master/CONTRIBUTING.md).

### Affected app version

1.1.1

### Affected Android/Custom ROM version

Android 9 to 11

### Affected device model

diverse

### How did you install the app?

F-Droid / IzzyOnDroid

### Steps to reproduce the bug

1. Open Fossify Gallery's main screen, make sure you are in `Show all folders contents` mode
2. In the three dots menu, open `Group by`, customize your grouping settings, press OK to confirm
3. In the three dots menu, open `Settings`, scroll down to find `Export settings`, tap there, note down and confirm the suggested filename
4. Reinstall Fossify Gallery or delete its data
5. Open Fossify Gallery again
6. In the three dots menu, open `Settings`, scroll down to find `Import settings`, tap there, select the noted file exported in step 3, and confirm
7. In 

### Expected behavior

The import should restore your customized grouping settings.

### Actual behavior

Grouping settings are still at their default values after the import.

### Screenshots/Screen recordings

![FossifyGallery_grouping_before_export](https://github.com/FossifyOrg/Gallery/assets/15276963/1e32c2eb-b9da-4488-90ea-387513b42a83)
![FossifyGallery_grouping_after_import](https://github.com/FossifyOrg/Gallery/assets/15276963/ad8a2902-1459-4727-9201-7d1045ab2746)

### Additional information

Although I haven't systematically tested export/import preservation with all options, there may be more that get lost. In the course of testing the grouping setting reported here, I've encountered a few times where the formerly activated setting `Show the Recycle Bin as the last item on the main screen` didn't get restored, either, but was "Off" after the import.

**Comment by GfEW** [comment]
14 months in, I'm a little puzzled regarding the label. Are settings not expected to be preserved by export-wipe-import cycles?

I obviously understand the need for priorisation - there are more severe bugs for sure, and it makes a lot of sense to differentiate severity by lables.

However, addressing the failure to reliably restore all settings as a _"feature request"_ feels odd. I'd rather think that issues like this one could be labeled more appropriately e. g. as `minor` `bug`.

**Comment by naveensingh** [comment]
Yep, let's call this a bug. Especially since the app already 'tries' to import/export some of these options.
